### PR TITLE
Address left open commends in #1391

### LINKED ISSUE #1392: Resolve compiler "var" ambiguation and use of "int" where we need "long".
This ticket is about using `var` to declare variables and address compiler's decision to use `int` (as inferred from code) instead of `long` as needed -> i.e. `var foo = 1;` (int) vs `var foo = 1L;` (long).

---

This code block was copied from another TTL part.
I will create another PR where I fix this, [that](https://github.com/logisim-evolution/logisim-evolution/pull/1391#discussion_r813025326) and [that](https://github.com/logisim-evolution/logisim-evolution/pull/1391#discussion_r812603085) in multiple files.

_Originally posted by @R3dst0ne in https://github.com/logisim-evolution/logisim-evolution/pull/1391#discussion_r813045331_

### PR DESCRIPTION
**Pull request by R3dst0ne**
Addresses things noted in [this](https://github.com/logisim-evolution/logisim-evolution/pull/1391#discussion_r813045331) comment.

---
> Can we have `p` replaced with more descriptive name please (multiple places)?

-- [comment](https://github.com/logisim-evolution/logisim-evolution/pull/1391#discussion_r813025326) is still unresolved.
I thought `pokerCoordinates` or short `pokerCoords` might be a fitting name for `p`. But I'm not 100% sure if the name is correct.

### CODE REVIEW

File: src/main/java/com/cburch/logisim/std/ttl/Ttl74161.java [hunk 1]
```diff
@@ -139,17 +139,16 @@ public void paintInternal(InstancePainter painter, int x, int y, int height, boo
...
-      for (var i = 0; i < 4; i++) {
-        final var isSetBitValue = (value & (1 << (3 - i))) != 0;
-        gfx.setColor(isSetBitValue ? trueColor : falseColor);
-        gfx.fillOval(x + 52 + i * 10, y + height / 2 - 4, 8, 8);
-        gfx.setColor(Color.WHITE);
-        GraphicsUtil.drawCenteredText(gfx, isSetBitValue ? "1" : "0", x + 56 + i * 10, y + height / 2);
-      }
-      gfx.setColor(Color.BLACK);
+    if (state == null) return;
+    long value = state.getValue().toLongValue();
```
Comment (BFH-ktt1) [inline src/main/java/com/cburch/logisim/std/ttl/Ttl74161.java:143]: final var ?

**Review by BFH-ktt1 — CHANGES_REQUESTED** [review]

**Comment by R3dst0ne** [comment]
Can someone, who is more familiar with the code, tell me whether my name ideas for `p` are correct?

**Comment by MarcinOrlowski** [comment]
@R3dst0ne I peek the changed files and see just single change to `var`. All the remaining changes are logic changes and for me these are out of scope for this PR if you say it fixed #1392 because for me it looks it does not not that.

**Comment by R3dst0ne** [comment]
@MarcinOrlowski
I should have labeled it more clearly.
It targets the things noted in [this](https://github.com/logisim-evolution/logisim-evolution/pull/1391#discussion_r813045331) comment (quoted in the description of #1392).

**Review by BFH-ktt1 — APPROVED** [review]

**Pull request merged**